.github/workflows/repo-sync.yml

```diff
@@ -0,0 +1,27 @@
+name: Repo Sync
```

Comment:
Please add comment that describes what this file is and how it operates.

Reply:
This Git hub action workflows to keep pairs of git repos in sync. It runs on a Schedule time set by the user in the script. Shortly after changes are made to the default branch of repo A, the Actions workflow runs on repo B and generates a pull request including the recent changes from repo A. If more changes are made to repo A before the pull request is merged, those changes will be added to the existing pull request. The same is true in the opposite direction: changes made to repo B will eventually get picked up by the workflow in repo A.

Reply:
Does this file need to be in the release or can it stay in develop? @SR-btc

Reply:
@jkrogers18 
We can remove this file, as this file is required only if they want to sync the repo for later release and it is not linked with any application features or deployments, so I am removing this file